src/style.css

```diff
@@ -2,0 +1,117 @@
+:root {
+  --main-text-color: #3d4e61;
+  --white-color: #fff;
+  --light-white-color: #f6f6f7;
+  --main-border-color: #e1e7ed;
+  --primary-font: 16px;
+  --small-font: 14px;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url(./fonts/Avenir-Book.ttf) format('ttf');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url(./fonts/Avenir-Heavy.ttf) format('ttf');
```

Comment:
Similar to the previous `@font-face` declaration, the format should be 'truetype' instead of 'ttf'. Use `format('truetype')`.